client/src/App.js

```diff
@@ -136,6 +150,11 @@ function App() {
     (list) => list.favorites.length !== 0
   );
 
+  const nextHandler = () => {
```

Comment:
I would probably give this function a more "speaking" name like "handleNextPage"

Reply:
Good catch!